examples/cosmodc2/Cluster_pipelines/CLClusterBinning-20deg2-CL.yml

```diff
@@ -1,13 +1,13 @@
 #this step depends on where you run
 #for CCin2p3
-# site:
-#     name: cc-parallel
-#     mpi_command: "mpirun -n"
+site:
+    name: cc-parallel
+    mpi_command: "mpirun -n"
 
 #for NERSC
-site:
-   name: cori-batch
-   image: ghcr.io/lsstdesc/txpipe-dev
+#site:
+#    name: cori-batch
```

Comment:
Is this still `cori-batch`?  Or `perlmutter-batch`?

Reply:
It should be `nersc-batch` now.